core/state_processor_test.go

```diff
@@ -794,3 +795,97 @@ func TestProcessVerkleContractWithEmptyCode(t *testing.T) {
 		}
 	}
 }
+
+func TestProcessVerkleBalanceOpcode(t *testing.T) {
```

Comment:
Another reminder to eventually do some helpers to avoid repetition in these tests.
Not urgent, but just saying.